rules/disagreeing-with-powerful-stakeholders/rule.md

```diff
@@ -36,7 +32,7 @@ Let's see what tips and tricks can be applied to ensure a smooth discussion.
 When you have a disagreement with a senior colleague, it's likely that you have different perspectives and understanding. Before you talk about the issue, it's pivotal to check you are on the same page about the issue and looking for a path forward. Here are some tips for doing so:
 
 1. Repeat your understanding of the issue
-2. Ask permission
+2. Ask permission to disagree
```

Comment:
From many comments in Discussions, which i agree with, I dislike the term 'permission'...

I'd probably prefer something like:

2. Ask if they are open to discuss a different approach